### DISCUSSION TITLE
Multiple Live Feed could you help me?

### DISCUSSION THREAD

**Discussion by MigzAcosta** [dgreif/ring#905] [Q&A]
so i have bought maybe around 10 ring cameras, and i want to view it on 1 screen all at the same time. i tried to use this oneand only manage to view 1 cam at a time. so maybe someone knows a way to view it all in the same time with live feeds

**Comment by dgreif** [comment]
You would have to build something custom to view them all at the same time. It's definitely possible, but would be a fair amount of work and is outside the scoped of what I'm trying to accomplish here. Good luck!
